daemon/images/image_list.go

```diff
@@ -69,8 +69,8 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 		}
 		// Resolve multiple values to the newest image,
 		// equivalent to ANDing all the values together.
-		if sinceFilter.Before(img.Created) {
-			sinceFilter = img.Created
+		if img.Created != nil && sinceFilter.Before(*img.Created) {
+			sinceFilter = *img.Created
```

Comment:
In the interest of highlighting the bits I'm the least sure about, this is one I _think_ I've got right, but am not positive.

Reply:
I think this is correct.